test/utils.jl

```diff
@@ -64,8 +64,8 @@ function build_libblastrampoline()
     cflags_add = needs_m32() ? "-m32" : ""
```

Comment:
```suggestion
    cflags_add = "-Werror" * needs_m32() ? " -m32" : ""
```
so that you don't need to add `-Werror` twice below